### PR TITLE
TS map They All Float added

### PR DESCRIPTION
**Pull request by fruestueck**
Added TS map: They All Float (originally by Westwood), Part of Issue #12518

2-Player map contains five tunnels and should be performance friendly.

During import of the original map the following unknown actor types got ignored:

> Ignoring unknown actor type: `ingrnlmp` (16 times)
> Ignoring unknown actor type: `neglamp` (3 times)
> Ignoring unknown actor type: `inblulmp` (3 times)
> Ignoring unknown lighting type: `Ground`
> Ignoring unknown lighting type: `Level`
> Ignoring unknown lighting type: `IonAmbient`
> Ignoring unknown lighting type: `IonRed`
> Ignoring unknown lighting type: `IonGreen`
> Ignoring unknown lighting type: `IonBlue`
> Ignoring unknown lighting type: `IonGround`
> Ignoring unknown lighting type: `IonLevel`

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -98,6 +98,7 @@ Also thanks to:
     * Max621
     * Max Ugrumov (katzsmile)
     * Michael Rätzel
+		* Michael Silber (frühstück)
```
Comment (Phrohdoh) [inline AUTHORS]: Please use spaces not tabs in this file.

Comment (fruestueck) [reply AUTHORS]: Sorry, will do better next time. That came from Tabs only in C# : )

Should i change it?

File: mods/ts/maps/float/map.yaml [hunk 2]
```diff
@@ -0,0 +1,664 @@
...
+	PlayerReference@Multi1:
+		Name: Multi1
+		Playable: True
+		Faction: Random
+		Enemies: Creeps
+
+Actors:
+	Actor0: jfish
+		Location: 39,5
+		Owner: Neutral
```
Comment (Mailaender) [inline mods/ts/maps/float/map.yaml]: Not sure if this is intentional as it will make them not attack the player. I would suggest `Owner: Creeps` instead.

**Comment by Mailaender** [comment]
Tunnels work fine. :white_check_mark:

File: mods/ts/maps/float/map.yaml (hunk 2 above)
Comment (fruestueck) [reply mods/ts/maps/float/map.yaml]: Ty, changed to Creeps. Didn't know if there are any neutral Creeps.

**Comment by chrisforbes** [comment]
This has no reviews?

Commits should be squashed.

**Comment by abcdefg30** [comment]
> This has no reviews?

> Tunnels work fine. :white_check_mark:

**Review by fruestueck — COMMENTED** [review]
Squashed all 4 commits

**Review by abcdefg30 — APPROVED** [review]

**Pull request merged**